src/Collection/CollectionTrait.php

```diff
@@ -89,9 +89,8 @@ public function reject(callable $c)
      */
     public function every(callable $c)
     {
-        $return = false;
+        $return = true;
```

Comment:
The variable `$return` could be eliminated altogether, it always has the value of `false`.